packages/connectivity/android/src/main/java/io/flutter/plugins/connectivity/ConnectivityPlugin.java

```diff
@@ -105,7 +129,7 @@ private void handleCheck(MethodCall call, final Result result) {
   private String checkNetworkType() {
     NetworkInfo info = manager.getActiveNetworkInfo();
```

Comment:
`NetworkInfo` is deprecated per documentation here: https://developer.android.com/reference/android/net/ConnectivityManager
So this line can be put into the legacy implementation as well. An non-deprecated method should be used to check if the network is connected.